{
  "data_type": "CVE",
  "data_format": "MITRE",
  "data_version": "4.0",
  "CVE_data_meta": {
    "ID": "CVE-2022-48340",
    "ASSIGNER": "cve@mitre.org"
  },
  "problemtype": {
    "problemtype_data": [
      {
        "description": [
          {
            "lang": "en",
            "value": "CWE-416"
          }
        ]
      }
    ]
  },
  "references": {
    "reference_data": [
      {
        "url": "https://github.com/gluster/glusterfs/issues/3732",
        "name": "https://github.com/gluster/glusterfs/issues/3732",
        "refsource": "MISC",
        "tags": [
          "Exploit",
          "Issue Tracking",
          "Patch",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/UE6K2DXP4QZVKP32Z7BSYDSRBL4H7JSE/",
        "name": "FEDORA-2023-5a54eea360",
        "refsource": "",
        "tags": []
      }
    ]
  },
  "description": {
    "description_data": [
      {
        "lang": "en",
        "value": "In Gluster GlusterFS 11.0, there is an xlators/cluster/dht/src/dht-common.c dht_setxattr_mds_cbk use-after-free."
      }
    ]
  }
}